risor.go

```diff
@@ -33,6 +34,61 @@ func WithGlobal(name string, value any) Option {
 	}
 }
 
+// WithoutGlobal opts out of a given global builtin or module. If the name can't
+// be resolved, this is a no-op. This does operate on nested modules.
+func WithoutGlobal(name string) Option {
+	return func(cfg *Config) {
+		parts := strings.Split(name, ".")
+		if len(parts) == 1 {
+			attrName := parts[0]
+			delete(cfg.Globals, attrName)
+			delete(cfg.DefaultGlobals, attrName)
+			return
+		}
+		// Find the named module
+		moduleName := parts[0]
+		var module *object.Module
+		if obj, ok := cfg.Globals[moduleName]; ok {
+			if m, ok := obj.(*object.Module); ok {
+				module = m
+			}
+		} else if obj, ok := cfg.DefaultGlobals[moduleName]; ok {
```

Comment:
if the same name is set on both `cfg.Globals` and `cfg.DefaultGlobals` it's enough to override/remove the first one? I think that works well for override but I'm not sure about the removal, but I'm also not sure if such edge case would even bee possible.

Reply:
Good call. It's possible.